src/services/mondomarket/index.jsx

```
@@ -264,15 +271,17 @@ export const MondoDetailsViewer = ({
   const { textColor } = useSelector((state) => state.theme);
   const { isLoading } = useSelector((state) => state.mondomarket);
+  const { user } = useSelector((state) => state.user);
 
   const [offerModal, setOfferModal] = useState(false);
   const [amount, setAmount] = useState("");
+  const [alertSetteled, setAlertSetteled] = useState(false);
 
   const handleBookmarkItem = async () => {
     try {
       const payload = {
         item_id: mondo?.id,
```

ESLint

[error] 'mondo.id' is missing in props validation

(react/prop-types)